strings/tips.txt

```diff
@@ -0,0 +1,189 @@
+Where the space map levels connect is randomized every round, but are otherwise kept consistent within rounds. Remember that they are not necessarily bidirectional!
+You can catch thrown items by toggling on your throw mode with an empty hand active.
+To crack the safe in the vault, use a stethoscope or thermal drill.
+You can climb onto a table by dragging yourself onto one. This takes time and drops the items in your hands on the table. Clicking on a table that someone else is climbing onto will knock them down.
```

Comment:
Unless this changed without me noticing (entirely possible), we don't drop items when climbing here.

Reply:
Fixed in next commit